### ISSUE TITLE
FPM closes if we want to apply the backup on Android 5.1.1

### ISSUE THREAD

**Issue by sflorean** [mozilla-mobile/fennec-profile-manager#32]
## Preconditions
- Firefox Profile Manager is installed.
- Backup1 is created. 

## Steps to reproduce
1. Open FPM. 
2. Swipe from left to right for Backup1.
3. Tap on the "Apply" button.
4. Tap on "Yes".

### Expected behavior
"The backup has been applied successfully, please check the app." message is displayed. 

### Actual behavior
FPM Nightly closes, and the following message is displayed on the device homepage: "Unfortunately, FPM Nightly has stopped." 

### Device information
* Android device: Samsung Galaxy Tab A 6 (Android 5.1.1).
* Build Firefox Profile Manager v1.0 for Fennec Release, Beta and Nightly.

**Comment by sflorean** [comment]
[logcat.txt](https://github.com/mozilla-mobile/fennec-profile-manager/files/3932594/logcat.txt)

**Comment by Mugurell** [comment]
```
12-06 16:30:07.342 9863-9863/org.mozilla.fpm_nightly E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.fpm_nightly, PID: 9863
    java.util.zip.ZipException: File too short to be a zip file: 0
        at java.util.zip.ZipFile.readCentralDir(ZipFile.java:376)
        at java.util.zip.ZipFile.<init>(ZipFile.java:163)
        at java.util.zip.ZipFile.<init>(ZipFile.java:119)
        at org.mozilla.fpm.utils.ZipUtils.extract(ZipUtils.java:65)
        at org.mozilla.fpm.data.BackupRepositoryImpl.deploy(BackupRepositoryImpl.kt:62)
        at org.mozilla.fpm.presentation.mvp.MainPresenter$applyBackup$1$1.invokeSuspend(MainPresenter.kt:37)
```

**Comment by sv-sdeiac** [comment]
Verified as fixed on FPM v1.0 Build 2 using Samsung Galaxy Tab A6 (Android 5.1).
